tests/Selenium2Config.php

```diff
@@ -62,6 +62,18 @@ public function skipMessage($testCase, $test): ?string
             return 'Checking status code is not supported.';
         }
 
+        if (
+            'Behat\Mink\Tests\Driver\Js\JavascriptTest' === $testCase
+            && 'testDragDropOntoHiddenItself' === $test
```

Comment:
Small hint: using something so allows IDEs and static code analysers to see this as a reference to the method (e.g. `[ctrl]`+`[click]`ing `testDragDropOntoHiddenItself` would open the method in PhpStorm):
```php
if ([JavascriptTest::class, 'testDragDropOntoHiddenItself'] === [$testCase, $test]) {
```

Reply:
Done.